draft-irtf-cfrg-vdaf.md

```diff
@@ -401,13 +401,13 @@ security considerations for VDAFs.
 
 05:
 
```

Comment:
Reviwer tip: Check out the branch and look the Change Log section in `git diff draft-irtf-cfrg-vdaf-06 draft-irtf-cfrg-vdaf.md`